turborepo-tests/integration/tests/ordered/github.t

```diff
@@ -33,6 +33,36 @@ because otherwise prysk interprets them as multiline commands
   Cached:    0 cached, 2 total
     Time:\s*[\.0-9]+m?s  (re)
   
+# Build as if we are in Github Actions with a task log prefix.
+  $ export GITHUB_ACTIONS=1
+  $ ${TURBO} run build --force --log-prefix="task"
```

Comment:
```suggestion
GITHUB_ACTIONS=1 ${TURBO} run build --force --log-prefix="task" --filter=util
```

- not blocking, but i try to set env vars just for the command so it doesn't leak into other tests in the file
- added a filter flag so you don't run into non-deterministic ordering output. Based on the fact that `util#build` is logged _after_ `my-app#build` in this test right now, I'm guessing they are running concurrently

Reply:
Should I apply this to the preceding test? Also if I'm not mistaken don't these `.t` files need to start with a `$` in order for the command to be executed in a shell?

Reply:
+1 to the `--filter`

Reply:
> do we care to have two tasks or is it ok to just test that a single task has the prefix?
single task is fine IMO

> Should I apply this to the preceding test?
generally, I wouldn't change unrelated, existing code. That has a tendency to balloon the diff and make git history harder to follow (not to mention it might cause unexpected issues and sink your time). In this case, 🤷🏾